recipes/uwsgi/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "2.0.12" %}
+
+package:
+  name: uwsgi
+  version: '{{ version }}'
+
+source:
+  fn: "uwsgi-{{ version }}.tar.gz"
+  url: "https://pypi.io/packages/source/u/uWSGI/uwsgi-{{ version }}.tar.gz"
+  md5: 1451cab954bad0d7d7429e4d2c84b5df
+  patches:
+    - darwin.patch   # [osx]
+    - darwin2.patch  # [osx]
+    # ^^ patch darwin2 should be able to be deleted on the next version pump
```

Comment:
Have these patches been submitted to upstream? Could we please have referenced to the relevant PRs?